### PR TITLE
Adds XPS example files and completes tutorial docs

### PR DESCRIPTION
**Pull request by sherjeelshabih**

### CODE REVIEW

File: docs/tutorial/converting-data-to-nexus.md [hunk 1]
```diff
@@ -48,13 +48,14 @@ pwd
 
 Once you have your files copied into the working directory. Your directory structure should look like this:
 ```
-- file1.vms
-- otherfile.vms
+├── main_data_file.sle
+├── metadata_file.slh
```
[the remark this answers is not in the document]
Comment (sherjeelshabih) [reply docs/tutorial/converting-data-to-nexus.md]: I was just on this. :D This should be resolved with the latest commit.

File: .github/workflows/build_docs.yml [hunk 2]
```diff
@@ -24,4 +24,11 @@ jobs:
...
-      - run: mkdocs gh-deploy --force
\ No newline at end of file
+      - run: mkdocs gh-deploy --force
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          folder: ./
+          branch: gh-pages
+          clean: true
```
Comment (sherjeelshabih) [inline .github/workflows/build_docs.yml:34]: This shouldn't be here. Removing it.

File: docs/tutorial/converting-data-to-nexus.md [hunk 3]
```diff
@@ -27,36 +23,41 @@ You will have
...
-We will use the [XPS Reader](https://github.com/FAIRmat-NFDI/pynxtools/tree/master/pynxtools/dataconverter/readers/xps) with Vamas (.vms) files as an example. 
+We will use the [XPS Reader](https://github.com/FAIRmat-NFDI/pynxtools/tree/master/pynxtools/dataconverter/readers/xps) with a [SpecsLabProdigy](https://www.specs-group.com/nc/specs/products/detail/prodigy/) file (.sle) as an example.
 
 #### Steps
 
-1. Download the example files from here:
-2. Install [pynxtools](https://github.com/FAIRmat-NFDI/pynxtools/tree/master?tab=readme-ov-file#installation)
-3. Verify you can run the ```dataconverter``` in a terminal window. Open a terminal with the Python environment where you installed ```pynxtools```. Then type the following:
+1. Download the example files from here: [Example files](https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2FFAIRmat-NFDI%2Fpynxtools%2Ftree%2Fdocumentation%2Fexamples%2Fxps)
+2. **Extract** the zip and copy the files in your current working directory. You can find the working directory by typing the following in your terminal:
```
Comment (lukaspie) [inline docs/tutorial/converting-data-to-nexus.md:31]: Maybe add how to extract the files from the zip from the console.

**Review by lukaspie — APPROVED** [review]
LGTM expect for one small comment

**Pull request merged**